docs/installation.rst

```diff
@@ -1,6 +1,44 @@
 Installing
 ==========
 
+TL;DR
+-----
+
+This is for people who are in a hurry and just want to get started. We advice
+you to read the full documentation below if you are new to Python or the Python
+packaging ecosystem.
+
+Regular install
+^^^^^^^^^^^^^^^
+
+Install with pixi::
+
+  pixi add imod
+
+Install with conda::
+
+  conda install imod
+
+Install with pip::
+
+  pip install imod
+
+Install beta release
```

Comment:
We should remove this once 1.0 is released. You could make a task on the board so that we don't forget that.